tests/yamllint.yml

```diff
@@ -3,6 +3,7 @@ extends: default
 rules:
   line-length:
     level: error
+    allow-non-breakable-inline-mappings: true
```

Comment:
What does this do, and is it something we should include in the meta_skeleton for future roles?

Reply:
It allows line-length to exceed the default maximum in non-breakable strings in inline mappings. In my case, it lets me put longish urls into a playbook without having to mangle them or do other trickery. This is for a specific downstream test scenario, though, so it's probably worth using only in that scenario. I do not think it's worth including in the meta_skeleton by default, but I do think it's good to know about.

tl;dr I'm going to remove this. :)